lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/generic/Launch.java

```diff
@@ -111,10 +111,8 @@ private static Configuration getClientConfiguration(final String[] args)
           .set(ClientConfiguration.ON_JOB_COMPLETED, JobClient.CompletedJobHandler.class)
           .set(ClientConfiguration.ON_JOB_FAILED, JobClient.FailedJobHandler.class)
           .set(ClientConfiguration.ON_RUNTIME_ERROR, JobClient.RuntimeErrorHandler.class)
-          //.set(ClientConfiguration.ON_WAKE_ERROR, JobClient.WakeErrorHandler.class )
           .build();
 
-      // TODO: Remove the injector, have stuff injected.
```

Comment:
I've asked the author of the code at https://issues.apache.org/jira/browse/REEF-521, they don't remember.